pkg/services/publicdashboards/api/api.go

```diff
@@ -55,27 +55,30 @@ func ProvideApi(
 
 // RegisterAPIEndpoints Registers Endpoints on Grafana Router
 func (api *Api) RegisterAPIEndpoints() {
-	auth := accesscontrol.Middleware(api.AccessControl)
 
+	// Public endpoints
 	// Anonymous access to public dashboard route is configured in pkg/api/api.go
 	// because it is deeply dependent on the HTTPServer.Index() method and would result in a
 	// circular dependency
 
-	// public endpoints
 	api.RouteRegister.Get("/api/public/dashboards/:accessToken", routing.Wrap(api.GetPublicDashboard))
 	api.RouteRegister.Post("/api/public/dashboards/:accessToken/panels/:panelId/query", routing.Wrap(api.QueryPublicDashboard))
 	api.RouteRegister.Get("/api/public/dashboards/:accessToken/annotations", routing.Wrap(api.GetAnnotations))
 
-	// List Public Dashboards
-	api.RouteRegister.Get("/api/dashboards/public", middleware.ReqSignedIn, routing.Wrap(api.ListPublicDashboards))
-
-	// Create/Update Public Dashboard
+	// Auth endpoints
+	auth := accesscontrol.Middleware(api.AccessControl)
 	uidScope := dashboards.ScopeDashboardsProvider.GetResourceScopeUID(accesscontrol.Parameter(":uid"))
-	api.RouteRegister.Get("/api/dashboards/uid/:uid/public-config",
+
+	// List public dashboards for org
+	api.RouteRegister.Get("/api/dashboards/public-dashboards", middleware.ReqSignedIn, routing.Wrap(api.ListPublicDashboards))
+
+	// Get public dashboard
+	api.RouteRegister.Get("/api/dashboards/uid/:dashboardUid/public-dashboard",
```

Comment:
From a REST perspective, all `public-dashboards` in the URL should be plural.

Reply:
Sure. let's do it